Packs/ThreatZone/Integrations/ThreatZone/README.md

```diff
@@ -0,0 +1,791 @@
+| mouse_simulation    | Enable mouse simulation during the analysis. Possible values are: true, false. Default false.                                                          | Optional     |
+| https_inspection    | Https inspection to read encrypted traffic. Possible values are: true, false. Default is false.                                                        | Optional     |
+| internet_connection | Enable internet connection during the analysis. Possible values are: true, false. Default is false.                                                    | Optional     |
+| raw_logs            | Raw logs. Possible values are: true, false. Default is false.                                                                                          | Optional     |
+| snapshot            | Snapshot. Possible values are: true, false. Default is false.                                                                                          | Optional     |
+
+#### Output
+
+
+| **Path**                 | **Type** | **Description**                                     |
+| -------------------------- | ---------- | ----------------------------------------------------- |
+| ThreatZone.Analysis.UUID | String   | UUID of the task created to analyze the submission. |
+| ThreatZone.Analysis.URL  | String   | URL of the task created to analyze the submission.  |
+
+### tz-static-upload-sample
+
+---
+
+Submit a file for static analysis.
+
+#### Base Command
+
+`tz-static-upload-sample`
+
+#### Input
+
+
+| **Argument Name** | **Description**                 | **Required** |
+| ------------------- | --------------------------------- | -------------- |
+| entry_id          | EntryID of the file to analyze. | True         |
+
+#### Output
+
+
+| **Path**                 | **Type** | **Description**                                     |
+| -------------------------- | ---------- | ----------------------------------------------------- |
+| ThreatZone.Analysis.UUID | String   | UUID of the task created to analyze the submission. |
+| ThreatZone.Analysis.URL  | String   | URL of the task created to analyze the submission.  |
+
+### tz-cdr-upload-sample
+
+---
+
+Submit a file to ThreatZone CDR.
+
+#### Base Command
+
+`tz-cdr-upload-sample`
+
+#### Input
+
+
+| **Argument Name** | **Description**                 | **Required** |
+| ------------------- | --------------------------------- | -------------- |
+| entry_id          | EntryID of the file to analyze. | True         |
+
+#### Output
+
+
+| **Path**                 | **Type** | **Description**                                     |
+| -------------------------- | ---------- | ----------------------------------------------------- |
+| ThreatZone.Analysis.UUID | String   | UUID of the task created to analyze the submission. |
+| ThreatZone.Analysis.URL  | String   | URL of the task created to analyze the submission.  |
+
+### tz-get-report
+
```

Comment:
this command are not defined in the yml file and not implemented in the py file

Please regenerate a README.md file.
To do that, please run the `demisto-sdk generate-docs` command (use the -e flag for using a commad_examples file) - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#generate-docs.